pom.xml

```diff
@@ -1621,6 +1632,8 @@
         <jackson.dataformat.yaml.version>2.9.9</jackson.dataformat.yaml.version>
         <javax.validation.version>1.1.0.Final</javax.validation.version>
         <snakeyaml.version>1.25</snakeyaml.version>
+        <!-- Prometheus Library versions-->
+        <prometheus.version>0.8.1</prometheus.version>
```

Comment:
We are adding this in multiple PRs now. I think we should coordinate and add this entry in one PR and get it merged before merging others.

Reply:
Let's merge this PR first.